### ISSUE TITLE
Sandcastle - New button

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium#1898]
Should Sandcastle have a `New` button that loads the Hello World example?

**Comment by 111CR0103** [comment]
I have attended this issue and created a 'New' Button in the cesium viewer app which when clicked redirects the user to 'Hello World' example. With the help of this code, I can even add this functionality to Sandcastle. Below image and code shows its implementation.

```javascript
<script>
        function home() {
            console.log('Hello');
            window.location.assign("http://127.0.0.1/cesium/Apps/HelloWorld.html");
        }
    </script>
```
![pageview](https://cloud.githubusercontent.com/assets/2352166/6523565/11d8c822-c417-11e4-87d6-f3ba4be19bfb.png)

**Comment by pjcozzi** [comment]
Thanks for the initial work, @111CR0103.  Our plan here is to put this in Sandcastle using the same style as the rest of the Sandcastle buttons.

**Comment by adi2412** [comment]
Mind if I work on this one?(Apologies if someone might be already working on this)

I'm guessing the button needs to be in the top toolbar, like so(New on the top left):

![screen shot 2015-03-07 at 8 01 45 pm](https://cloud.githubusercontent.com/assets/2319795/6541671/d12258ea-c504-11e4-9e91-06c9fcafb124.png)

**Comment by mramato** [comment]
@adi2412 Yes, that is correct.  Feel free to open a pull request when you are ready.  @111CR0103 I know you were looking at this too but we'll take whomever opens a good pull request with the change first.

If one of you want to work on #2178 too, that would be cool.

In either case, when you open a PR, make sure you also sign and email us a Contributor License Agreement. The details for this are in [Contributing.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md) You most likely want the [individual CLA](http://www.agi.com/licenses/individual-cla-agi-v1.0.txt)  We need a CLA in order to merge any code from you into Cesium.

**Comment by 111CR0103** [comment]
Adding a button on the top left is easy. Just add the following piece of code inside the `div` with id `#toolbar` and attached the above mentioned Javascript.
````html
<button class="cesium-button cesium-button-toolbar">New</button>
````
adding the `.cesium-button` and `.cesium-toolbar-button` classes to the button achieves what I have done already and what you are saying. But I think adding it to the top-right toolbar with other buttons will be more appealing. I suggest you to host the entire cesium app locally and use chrome developer tools (that's how I am trying to do it) or you can use firebug debugger for firefox.

**Comment by emackey** [comment]
This issue should be considered "on hold" until after #2178 is resolved.
